### DISCUSSION TITLE
Freenom好像停止運行了

### DISCUSSION THREAD

**Discussion by peteryang1756** [luolongfei/freenom#214] [General]
好像是因為被Meta告https://krebsonsecurity.com/2023/03/sued-by-meta-freenom-halts-domain-registrations/
有人知道為何現在無法注冊呢

**Comment by luolongfei** [comment]
现在看起来是不能注册新域名了，但是之前的域名依然可以续期。

**Comment by Angel0726** [comment]
哪里还可以注册免费的

**Reply by peteryang1756** [reply]
Eu.org
